controllers/get-endpoints.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchEndpoints } = require("../models/fetch-endpoints.model");
+
+exports.getEndpoints = (request, response, next) => {
+  fetchEndpoints()
```

Comment:
In your controller, it may be easier to require in the endPoints json and send it back - this results in less code